bindings/matlab/+iDynTreeWrappers/generalizedBiasForces.m

```diff
@@ -0,0 +1,36 @@
+function h = generalizedBiasForces(KinDynModel)
```

Comment:
Sorry for bothering, but can you add copyright information to this files? Something like:
~~~
% Copyright (C) 2019 Istituto Italiano di Tecnologia (IIT). All rights reserved.
% This software may be modified and distributed under the terms of the
% GNU Lesser General Public License v2.1 or any later version.
~~~